httpapi/userapi/transfer_details_api_test.go

```diff
@@ -0,0 +1,87 @@
+// +build unit
+
+package userapi
+
+//func TestHandler_CreateDocument(t *testing.T) {
```

Comment:
Why commented out ?

Reply:
testing is a WIP, just finished the functionality first for PR review and for rasvan to start experimenting with the API calls